scripts/optimize_registration.py

```diff
@@ -134,7 +134,7 @@ async def train_coro(
                 moving_path,
                 fixedmask_path=brainmask_path,
                 **_kwargs,
-            )
+            ).cmdline
```

Comment:
The `generate_command` function has been modified to return the nipype interface and that way unify the antsRegistration command line generation in a single function.